mlrun/api/db/sqldb/db.py

```diff
@@ -1544,10 +1544,14 @@ def _update_db_record_from_object_dict(
         db_object.name = common_object_dict["metadata"]["name"]
         updated_datetime = datetime.now(timezone.utc)
         db_object.updated = updated_datetime
+        db_object.created = common_object_dict["metadata"].pop(
+            "created", None
+        ) or datetime.now(timezone.utc)
```

Comment:
We don't want to allow someone to change the object's creation time after it was created, this allows it IIUC

Reply:
hmm... yes, good catch - this method is actually called in creating new objects as well (an empty object is created and then this function fills it up) so it's needed, but I'll add a check to verify it only changes the `created` timestamp if it's `None` before this runs.